html/captive-portal/lib/captiveportal/PacketFence/Controller/DeviceRegistration.pm

```diff
@@ -61,15 +61,15 @@ sub index : Path : Args(0) {
             # Register device
             $c->forward('registerNode', [ $pid, $device_mac, $device_type ]);
             unless ($c->has_errors) {
-                $c->stash(status_msg  => i18n_format("The MAC address %s has been successfully registered.", $device_mac));
+                $c->stash(status_msg  => [ "The MAC address %s has been successfully registered.", $device_mac ]);
                 $c->detach('landing');
             }
         } else {
-            $c->stash(txt_auth_error => i18n_format("Please verify the provided MAC address."));
+            $c->stash(txt_auth_error => "Please verify the provided MAC address.");
         }
     }
     # User is authenticated so display registration page
-    $c->stash(template => 'device-registration/registration.html');
+    $c->stash(title => "Registration", template => 'device-registration/registration.html');
```

Comment:
i18n or directly in the template IMO

The configuration will get heavy for something that is never changed